### DISCUSSION TITLE
IPERF3 3.17.1 build on Ubuntu 24.04 still shows 3.16

### DISCUSSION THREAD

**Discussion by pdindak** [esnet/iperf#1727] [Q&A]
This is the only iperf3 on the server
**~/Downloads/iperf-3.17.1$ which iperf3**
/usr/local/bin/iperf3
**~/Downloads/iperf-3.17.1$ iperf3 -v**
iperf 3.16 (cJSON 1.7.15)
Linux ENIAPP-RP1 6.8.0-1006-raspi #6-Ubuntu SMP PREEMPT_DYNAMIC Mon Jun 17 11:30:51 UTC 2024 aarch64
Optional features available: CPU affinity setting, IPv6 flow label, SCTP, TCP congestion algorithm setting, sendfile / zerocopy, socket pacing, authentication, bind to device, support IPv4 don't fragment, POSIX threads

Did configure, make install and it's in /usr/local/bin, not anywhere else on the server.

echo $PATH
/usr/local/sbin:/usr/local/bin: etc.........  
Nothing in /usr/local/sbin either
How to get the 3.17.1 I compiled to be the one called, not 3.16?

**Comment by davidBar-On** [comment]
I had the same problem and it seems that it was solved by running `make clean-libtool` before `make install`. (I don't know why that seems to solve the problem - maybe a conflict between the two versions build environment.)

I write "seem" above since after several failures I restarted my computer (for Windows updates) and only than did the `make clean-libtool`.  It may be that somehow the restart solved the problem, so if the `make clean-libtool`  doesn't help, suggest to try restarting the computer.

**Comment by pdindak** [comment]
Thanks for the suggestions! Still after the make clean-libtool and make install and a restart, still have the issue :-(
